.github/scripts/backport/main.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -e
+
+# Setting up Committer identity.
+git config user.email "noreply@github.com"
+git config user.name "GitHub Actions"
+
+# Get backport label from all the labels of the PR
+backport_label=""
+branch_name=""
+for label in "${LABELS[@]}"; do
+    echo "each label - $label"
+    if [[ $label == "backport"* ]]; then
+        backport_label=$label
+        branch_name=${label#backport }
+        break
+    fi
+done
+
+# Exit if backport label not found
+echo "Label Name: $backport_label"
+if [ -z "$backport_label" ]; then
+    echo "Backport label not found. Exiting."
+    exit 1
```

Comment:
Should it return `exit 1` or `exit 0` when backport label is not present?
As this script runs whenever a PR is closed, and the backport label is present, then the rest of the script will run; otherwise, it will fail.
cc @janosdebugs @cam72cam

Reply:
exit 0